projects/wave-core-new/src/lib/operators/dialogs/expression-operator/expression-operator.component.ts

```diff
@@ -200,9 +203,9 @@ export class ExpressionOperatorComponent implements AfterViewInit, OnDestroy {
                                 expression,
                                 output_type: dataType.getCode(),
                                 // TODO: make this configurable once units exist again
```

Comment:
Make it configurable via input or add a todo for this. You have to decide this by estimating the time this takes.

Reply:
ddd46a6d6a7bf0c25fcc0dd608c5f31c6f62ea82